packages/lesswrong/lib/collections/typingIndicators/views.ts

```diff
@@ -0,0 +1,17 @@
+import TypingIndicators from "./collection";
+import { ensureIndex } from '../../collectionIndexUtils';
+
+declare global {
+  interface TypingIndicatorsViewTerms extends ViewTermsBase {
+    view?: TypingIndicatorsViewName,
+    documentId?: string
+  }
+}
+
+TypingIndicators.addView("typingIndicatorsForPost", function (terms: TypingIndicatorsViewTerms) {
+  return {
+    selector: {documentId: terms.documentId},
+    options: {limit: 1}
+  }
+})
+ensureIndex(TypingIndicators, { documentId: 1 })
```

Comment:
Remove this index, it's mostly redundant with the one [here](https://github.com/ForumMagnum/ForumMagnum/pull/7857/files#diff-e0ab4ceed1c2f1705c22eed975841c262036e75021df3b99d43427304e6b3a1bR42)